viewing_party/party.py

```diff
@@ -1,23 +1,156 @@
 # ------------- WAVE 1 --------------------
-
+import copy
+'''
+This function takes in 3 values: the title, genre, and rating of a movie. It uses these three parameters
+and creates a dictionary that includes the title, genre, and rating for the movie. Returns none if any of
+the parameters are "falsy".
+'''
 def create_movie(title, genre, rating):
-    pass
-
-# -----------------------------------------
-# ------------- WAVE 2 --------------------
-# -----------------------------------------
-
-
-# -----------------------------------------
-# ------------- WAVE 3 --------------------
-# -----------------------------------------
-
-        
-# -----------------------------------------
-# ------------- WAVE 4 --------------------
-# -----------------------------------------
-
-# -----------------------------------------
-# ------------- WAVE 5 --------------------
-# -----------------------------------------
-
+    if not title or not genre or not rating:
+        return None
+    movie = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    return movie    
+'''
+This function takes in 2 values, a dictionary called user_data, and a dictionary called movie. It adds the movie
+as a value to the user_data under the key "watched", which is a list. 
+'''
+def add_to_watched(user_data, movie):
+    user_data_copy = copy.deepcopy(user_data)
+    user_data_copy["watched"].append(movie)
+    return user_data_copy
+'''
+This function takes in 2 values, a dictionary called user_data, and a dictionary called movie. It adds the movie
+as a value to the user_data under the key "watchlist", which is a list. 
+'''
+def add_to_watchlist(user_data, movie):
+    user_data_copy = copy.deepcopy(user_data)
+    user_data_copy["watchlist"].append(movie)

```

Comment:
[suggestion] When working with dictionary keys (which can be typo prone) it's sometimes helpful to make constants that our editor is better able to suggest for us. If we had

```python
KEY_WATCHLIST = "watchlist"
```

defined elsewhere, we could write

```python
    user_data_copy[KEY_WATCHLIST].append(movie)
```

We might get better suggestions from our editor, reducing typos. If in the future a key ever needed to change, we'd also only need to change it in one place. Refactoring shortcuts can also help, in a good editor, but sometimes we can help things along ourselves!